vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/ImagesSearchFullApiController.java

```diff
@@ -0,0 +1,80 @@
+package edu.asu.diging.vspace.web.staff.api;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.util.StringUtils;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.IVSImage;
+import edu.asu.diging.vspace.core.model.ImageCategory;
+import edu.asu.diging.vspace.core.services.IImageService;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+@Controller
+public class ImagesSearchFullApiController {
+
+    @Autowired
+	private IImageService imageService;
+
+    @Autowired
+	private ISpaceManager spaceManager;
+    
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @RequestMapping("/staff/images/search/full")
+	public String imageSearchFull(Model model,
+	@RequestParam(value = "searchText", required = false) String searchTerm) {
+    	return String.format("redirect:/staff/images/search/full/1?searchText=%s",(searchTerm==null?"":searchTerm));
+    }
+    
+    @RequestMapping("/staff/images/search/full/{page}")
+	public String imageSeacrhDescription(@PathVariable(required = false) String page,
```

Comment:
if `page` is not required, why do you have an extra controller method?

Reply:
have used only one controller method that is the second one. Removed the first controller method as it was not necessary and made the required changes